src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6986,4 +6986,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_LineContainsDifferentWhitespace" xml:space="preserve">
     <value>Line contains different whitespace than the closing line of the raw string literal: '{0}' versus '{1}'</value>
   </data>
+  <data name="ERR_UnexpectedMemberInTopLevelMain" xml:space="preserve">
+    <value>A top-level entry point cannot directly contain members such as fields, properties, and methods</value>
```

Comment:
The `methods` here feels a bit misleading. It can contain methods, namely local functions. Perhaps `instance methods` would be more clear? I'm open to other wordings as well.

Reply:
`instance methods` is confusing, I think. It could be interpreted as "static methods are okay". We may use the terminology from `MethodKind` enum and say `ordinary methods`, do you think this is good and less confusing?

Reply:
Ordinary methods sounds good to me.